src/main/java/net/sf/jabref/gui/maintable/MainTable.java

```diff
@@ -499,11 +499,14 @@ public void setSelected(int row) {
     }
 
     public int findEntry(BibEntry entry) {
-        return model.getTableRows().indexOf(entry);
-    }
-
-    public int findLastEntry(BibEntry entry) {
-        return model.getTableRows().lastIndexOf(entry);
+        EventList<BibEntry> tableRows = model.getTableRows();
```

Comment:
I need to find the position of the given entry in the model.
With the statement you described you would get an Optional of an Entry.